pjrt-plugin/iree/integrations/pjrt/common/compiler.cc

```diff
@@ -69,6 +69,28 @@ class InprocessCompilerJob : public CompilerJob {
     return std::string(cstr);
   }
 
+  void EnableCrashDumps(
+      ArtifactDumper::Transaction* artifact_transaction) override {
+    if (crash_dump_transaction_) return;
+    crash_dump_transaction_ = artifact_transaction;
+    ireeCompilerInvocationSetCrashHandler(
+        inv_, /*genLocalReproducer=*/false,
```

Comment:
We may want to make this a env variable later given how useful it is for nested failures.

Reply:
Yeah, didn't want to go crazy yet as there are a few knobs we ultimately want (among them a mode to enable artifacts but just for crashes).